sources/packages/web/src/views/student/application-offering-change-request/ApplicationOfferingChangeFormView.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Financial Aid Application"
+        sub-title="View Request"
+        :routeLocation="goBackRouteParams"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestStatus ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithStudent
+          "
+        >
+          <v-btn
+            v-if="showActionButtons"
+            color="primary"
+            variant="outlined"
+            @click="declineApplicationOfferingChangeRequest"
+            >Decline change</v-btn
+          >
+          <v-btn
+            v-if="showActionButtons"
+            color="primary"
+            class="ml-2"
+            @click="approveApplicationOfferingChangeRequest"
+            >Allow change</v-btn
+          >
+        </template>
+      </header-navigator>
+    </template>
+    <template #tab-header>
+      <v-tabs stacked color="primary"
+        ><v-tab
+          v-for="item in items"
+          :key="item.label"
+          :to="item.command()"
+          :ripple="false"
+        >
+          <span class="mx-2 label-bold"> {{ item.label }} </span>
```

Comment:
why do we need an `mx-2`, And I prefer to  add the span inside a `div`

Reply:
I have removed the margin. 

The span is inside a wrapper provided by vue. It is logically placed as a div > a > span. Here, `div `and `a` are vue provided wrappers. Screenshot below:

<img width="1906" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/48aa1d05-f23a-47b6-99a4-87781099f1e7">

I was thinking that it might add redundant html elements since there is already an existing `div` wrapper. 

@dheepak-aot @andrepestana-aot @guru-aot suggestions?

Reply:
For the benefit of time, I would say check with reviewer if that is not blocker and move on. Otherwise try to follow the convention which is already in place.

Reply:
@ann-aot  I strongly feel that using a `span` element directly inside a `v-tab` element is correct, so as to avoid using too many unnecessary `divs`. Below is a screenshot from the official vue webpage for the same:

<img width="1315" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/aee35cce-90d8-45be-9b0b-3275455cbc65">